codecs/jxl/dec/jxl_dec.cpp

```diff
@@ -1,31 +1,77 @@
 #include <emscripten/bind.h>
 #include <emscripten/val.h>
 
-#include <jpegxl/decode.h>
+#include <jxl/decode.h>
+#include "lib/jxl/color_encoding_internal.h"
```

Comment:
I'm assuming those will become public APIs over time? Seems a bit dangerous to rely on internals, but I suppose as long as we use a fixed commit SHA and don't attempt to upgrade, it's fine.

Reply:
It's not a public API, but it will be removed once the API functions are actually fixed to behave as intended :)